fhir/resources/core/fhirabstractmodel.py

```diff
@@ -66,6 +63,11 @@ class FHIRAbstractModel(BaseModel, abc.ABC):
     """Abstract base model class for all FHIR elements."""
 
     resource_type: str = ...  # type: ignore
+    model_config = ConfigDict(
+        extra="forbid",
+        populate_by_name=True,
+        validate_assignment=True,
+    )
```

Comment:
this replaces the `class Config: ...` approach in pydantic v2

Reply:
@nazrulworld @bwalsh the previously supported `json_loads` and `json_dumps` config parameters were removed in pydantic v2